corehq/apps/locations/templates/locations/manage/locations.html

```diff
@@ -124,15 +129,22 @@
                 {% if show_inactive %}
                 <a class="loc_edit btn btn-primary"
                     data-bind="click: function(data, event) { unarchive_loc(event.currentTarget, uuid()) }">
-                    Unarchive
+                    {% trans "Unarchive" %}
```

Comment:
I got tired of doing this by hand.  Vim users:
```
vnoremap <leader>djt "vda{% trans "" %}<Esc>bb"vp
```
Then select the string and hit `<leader>djt` (or whatever you wanna bind it to)